tests/xtro-sharpie/common-Foundation.ignore

```diff
@@ -802,7 +803,6 @@
 !unknown-type! NSObject bound
 
 ## Used 'INSFastEnumeration' placeholer (just so we can start flagging things)
```

Comment:
1. The same change is needed in the api-annotations-dotnet subdirectory.
2. You moved the line in this file in the end, so there's a dangling comment here now.

Reply:
Funny thing, locally it did not complain about the api-annotations-dotnet